tests/integration/test_charm.py

```diff
@@ -152,152 +146,18 @@ async def does_minio_bucket_exist(bucket_name, ops_test: OpsTest):
-    await ops_test.model.add_relation(istio_pilot, CHARM_NAME)
-
-    await ops_test.model.wait_for_idle(
-        [istio_gateway],
-        status="waiting",
-        timeout=600,
-    )
-
-    # Patch the istio-gateway Role so that it can access it's own configmap
-    # This can be removed when we move to the sidecar istio v1.11 charm
-    lightkube_client = Client(
-        namespace=ops_test.model_name,
-    )
-
-    await ops_test.model.set_config({"update-status-hook-interval": "15s"})
-    istio_gateway_role_name = "istio-gateway-operator"
-
-    new_policy_rule = PolicyRule(verbs=["*"], apiGroups=["*"], resources=["*"])
-    this_role = lightkube_client.get(Role, istio_gateway_role_name)
-    this_role.rules.append(new_policy_rule)
-    lightkube_client.patch(Role, istio_gateway_role_name, this_role)
-
-    sleep(50)
-    await ops_test.model.set_config({"update-status-hook-interval": "5m"})
-
-    await ops_test.model.wait_for_idle(status="active")
-
-
-@pytest.fixture
-@pytest.mark.asyncio
-async def url_with_ingress(ops_test: OpsTest):
-    status = await ops_test.model.get_status()
-    url = f"http://{status['applications']['istio-gateway']['public-address']}.nip.io/mlflow/"
-    yield url
-
-
-@pytest.fixture
-@pytest.mark.asyncio
-async def url_without_ingress(ops_test: OpsTest):
-    status = await ops_test.model.get_status()
-    unit_name = ops_test.model.applications[CHARM_NAME].units[0].name
-    url = f"http://{status['applications'][CHARM_NAME]['units'][unit_name]['address']}:5000"
-    yield url
-
-
-@pytest.mark.assertions
-@pytest.mark.parametrize(
-    "url", [lazy_fixture("url_without_ingress"), lazy_fixture("url_with_ingress")]
-)
-@pytest.mark.asyncio
-async def test_access_dashboard(request, url):
-    options = Options()
-    options.headless = True
-    options.log.level = "trace"
-    max_wait = 20  # seconds
-
-    kwargs = {
-        "options": options,
-        "seleniumwire_options": {"enable_har": True},
-    }
-
-    with webdriver.Firefox(**kwargs) as driver:
-        wait = WebDriverWait(driver, max_wait, 1, (JavascriptException, StopIteration))
-        for _ in range(60):
-            try:
-                driver.get(url)
-                wait.until(
-                    expected_conditions.presence_of_element_located(
-                        (By.CLASS_NAME, "experiment-view-container")
-                    )
-                )
-                break
-            except WebDriverException:
-                sleep(5)
-        else:
-            driver.get(url)
+async def test_can_create_experiment_with_mlflow_library(ops_test: OpsTest):
+    config = await ops_test.model.applications[CHARM_NAME].get_config()
+    url = f"http://localhost:{config['mlflow_nodeport']['value']}"
+    client = MlflowClient(tracking_uri=url)
+    response = requests.get(url)
+    client.create_experiment(TEST_EXPERIMENT_NAME)
+    all_experiments = client.search_experiments()
+    assert response.status_code == 200
```

Comment:
can we move this assertion up to the `response = ...` call?  If this gives `!=200`, the client calls don't matter